webpack.config.js

```diff
@@ -95,7 +100,26 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        use: ["style-loader", "css-loader"]
+        include: [
+          path.resolve(__dirname, "./pages"),
+          path.resolve(__dirname, "node_modules/patternfly"),
+          path.resolve(__dirname, "node_modules/@patternfly/patternfly"),
+          path.resolve(__dirname, "node_modules/@patternfly/react-styles/css"),
+          path.resolve(__dirname, "node_modules/@patternfly/react-core/dist/styles/base.css"),
+          path.resolve(__dirname, "node_modules/@patternfly/react-core/dist/esm/@patternfly/patternfly"),
+          path.resolve(__dirname, "node_modules/@patternfly/react-core/node_modules/@patternfly/react-styles/css")
+        ],
+        use: [MiniCssExtractPlugin.loader, "css-loader"]
+      },
+      {
+        test: /\.(svg|ttf|eot|woff|woff2)$/,
```

Comment:
Just as an fyi, here's an example of how these file types are handled in the patternfly-react-seed repo:
https://github.com/patternfly/patternfly-react-seed/blob/master/webpack.common.js#L32